src/firebase-messaging-service.service.ts

```diff
@@ -0,0 +1,102 @@
+import { Injectable } from "@angular/core";
+import { firebaseConfig } from "./environments/environment";
+import firebase from "firebase/compat/app";
+import "firebase/compat/messaging";
+import { getMessaging, getToken, onMessage } from "firebase/messaging";
+import { Logging } from "app/core/logging/logging.service";
+import { HttpClient } from "@angular/common/http";
+
+@Injectable({
+  providedIn: "root",
+})
+export class FirebaseNotificationService {
```

Comment:
Let's call this only "NotificationService", in case we ever switch the underlying system away from Firebase (and also use general method names). Or is this flow so specific and different from other frameworks that it cannot really be generalized?

Can you also add a short JsDoc above the service class that describes the role/purpose of this service?

Reply:
Sure, I will create the JsDoc for this service class.

Reply:
I will create a separate PR for this JsDoc for this service class.

Reply:
no, code documentation is part of implementation. Should be in this PR only, please. It literally just needs one sentence

Reply:
Okay, sure.

Reply:
I have added the JsDoc in the `notification.service.ts` file.

Reply:
Looks great :+1: